kfs-core/src/main/java/edu/arizona/kfs/gl/businessobject/EntryBo.java

```diff
@@ -0,0 +1,397 @@
+package edu.arizona.kfs.gl.businessobject;
+
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kuali.kfs.gl.businessobject.Entry;
+import org.kuali.kfs.sec.SecConstants;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.businessobject.AccountingLine;
+import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.kew.routeheader.DocumentRouteHeaderValue;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+import org.kuali.rice.krad.service.BusinessObjectService;
+
+import edu.arizona.kfs.sys.KFSPropertyConstants;
+
+/**
+ * This class is to .
+ *
+ * @author Adam Kost <kosta@email.arizona.edu> with some code adapted from UCI
+ */
+
+@SuppressWarnings("deprecation")

```

Comment:
About this EntryBo name, I think it could be prone to be confusing. We already have an Entry BO. Perhaps a name indicating your intent would be more apropos, something to indicate it is the "light", "thin", or "minimum" version w/out any of the reference objects. I just don't want future devs accidentally thinking this is the core Entry object.